cardano-cli/test/Test/Cli/Pioneers/Exercise2.hs

```diff
@@ -36,6 +36,9 @@ prop_createTransaction = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
   void $ execCardanoCLI
     [ "transaction", "build-raw"
     , "--tx-in", "91999ea21177b33ebe6b8690724a0c026d410a11ad7521caa350abdafa5394c3#0"
+    , "--auxiliary-script-file", "test/data/golden/shelley/multisig/scripts/all"
+    , "--tx-in", "91999ea21177b33ebe6b8690724a0c026d410a11ad7521caa350abdafa5394c3#0"
+    , "--auxiliary-script-file", "test/data/golden/shelley/multisig/scripts/all"
```

Comment:
Did we want to use `--auxiliary-script-file` here and not `--tx-in-script-file`?

Reply:
I intentionally added `--auxiliary-script-file` just to test that it works here.